### PR TITLE
Add test for provenance that covers multiple subjects

### PR DESCRIPTION
**Pull request by jul-sh**
The existing tests for wildcards resolves to a single file. This adds a test for multiple files by completing `.github/configs-docker/multi-config.toml` and running it in CI.

### CODE REVIEW

**Comment by jul-sh** [comment]
@tiziano88 requested to be cc'ed

**Review by laurentsimon — APPROVED** [review]
Thanks. Let me merge and trigger a run to see if it works

**Comment by laurentsimon** [comment]
Running in https://github.com/slsa-framework/example-package/actions/runs/8562087181

File: .github/configs-docker/multi-config.toml [hunk 1]
```diff
@@ -1 +1,13 @@
-# Produces multiple artifacts, in nested paths.
\ No newline at end of file
+# Produces multiple artifacts, in nested paths.
+command = [
+  "touch",
+  "one.multi",
+  "&&",
```
Comment (tiziano88) [inline .github/configs-docker/multi-config.toml:5]: how would this work? I don't believe there is a shell involved (but maybe I am missing something)

Comment (tiziano88) [reply .github/configs-docker/multi-config.toml:5]: (actually maybe there is, I see bash mentioned in the Docker command)

Comment (jul-sh) [reply .github/configs-docker/multi-config.toml:5]: we should probably update it to invoke "/bin/sh -c"

**Comment by laurentsimon** [comment]
The run failed in https://github.com/slsa-framework/example-package/actions/runs/8562087181/job/23464658395. PTAL

File: .github/configs-docker/multi-config.toml (hunk 1 above)
Comment (jul-sh) [reply .github/configs-docker/multi-config.toml:5]: perhaps we can spin the wheel of fortune once more with https://github.com/slsa-framework/example-package/pull/347. 

I can investigate more tomorrow also.

**Pull request merged**